### ISSUE TITLE
There is no option for set accuracy level high for user current location.

### ISSUE THREAD

**Issue by ankit-vaghela** [amay077/Xamarin.Forms.GoogleMaps#612]
### VERSIONS

* Xamarin.Forms.GoogleMaps - 3.0.0-beta3
* Xamarin.Forms - 2.5.0.280555
* 

### PLATFORMS

- [ ] Android
- [] iOS

### ACTUAL BEHAVIOR

https://developer.xamarin.com/api/type/Android.Locations.Accuracy/

Is there any option which is use to set accuracy level high for users or accuracy should be with in meter ? See above link of xamarin website. I want to set that in my app using your lib.

**Comment by amay077** [comment]
Yes, Xamarin.Forms.GoogleMaps does not support Android.Locations.Accuracy.
Because this is wrapper of [Google Maps SDK for Android](https://developers.google.com/maps/documentation/android-sdk/intro).
